lib/data.js

```diff
@@ -0,0 +1,15 @@
+import base from 'taskcluster-base';
+
+var data = {} = module.exports;
+
+data.SecretEntity = base.Entity.configure({
+    version:          1,
+    signEntities:     true,
+    partitionKey:     base.Entity.keys.ConstantKey('secrets'),
+    rowKey:           base.Entity.keys.StringKey('name'),
+    properties: {
+      name:           base.Entity.types.String,
+      secret:         base.Entity.types.EncryptedJSON,
+      expires:        base.Entity.types.Date
+    }
+});
```

Comment:
Totally nit comment you don't have to do this...
```js
let _ = require('lodash');

let SecretEntity = base.Entity.configure({
    version:          1,
    signEntities:     true,
    partitionKey:     base.Entity.keys.ConstantKey('secrets'),
    rowKey:           base.Entity.keys.StringKey('name'),
    properties: {
      name:           base.Entity.types.String,
      secret:         base.Entity.types.EncryptedJSON,
      expires:        base.Entity.types.Date
    }
});

exports.SecretEntity = SecretEntity;

/** Return JSON representation of the secret */
SecretEntity.prototype.json = function() {
  return {
    secret: _.cloneDeep(this.secret),,
    expires: this.expires.toJSON()
  };
};

```

Don't put too much logic into method on entities... but it's nice to have a little for json formatting, then you can do:

```js
let data = require('./data');

let Secret = SecretEntity.setup(...);

// in some api method:
let secret = Secret.load({name: ...})
res.reply(secret.json());
```